code/modules/multiz/basic.dm

```diff
@@ -37,6 +37,8 @@ var/list/z_levels = list()// Each bit re... haha just kidding this is a list of
 	return HasBelow(turf.z) ? get_step(turf, DOWN) : null
 
 /proc/GetConnectedZlevels(z)
+	if(z in using_map.map_levels)
+		return using_map.get_map_levels(z, FALSE) // Connected z levels aren't necessarily attached by multi-z, using_map shound know the details
```

Comment:
Yes, they most certainly are!  Call using_map.get_map_levels() if you want get_map_levels, the `GetConnectedZlevels()` proc is strictly for things that are multi-z connected.